doc/changes/latest.inc

```diff
@@ -45,6 +45,8 @@ Enhancements
 
 - Add projections when printing a :class:`mne.Info` in the notebook (:gh:`9403` by `Alex Gramfort`_)
 
+- Add label methods `mne.Label.compute_area` and `mne.Label.distances_to_edge` (:gh:`9424` by `Eric Larson`_)
```

Comment:
```suggestion
- Add label methods :meth:`mne.Label.compute_area` and :meth:`mne.Label.distances_to_edge` (:gh:`9424` by `Eric Larson`_)
```
?

Reply:
No need, our default role takes care of it. Most PRs lately haven't included these. I think @drammock has moved away from including them